### DISCUSSION TITLE
Is there a roadmap and governance system for Organic Maps?

### DISCUSSION THREAD

**Discussion by TheAdventurer64** [organicmaps/organicmaps#3345] [Q&A]
Are [ROADMAP](https://github.com/organicmaps/organicmaps/blob/master/docs/ROADMAP.md) and [GOVERNANCE](https://github.com/organicmaps/organicmaps/blob/master/docs/GOVERNANCE.md) up to date? Aside from the documentation, there hasn't been much said on either of them.

**Comment by rtsisyk** [comment]
> Are [ROADMAP](https://github.com/organicmaps/organicmaps/blob/master/docs/ROADMAP.md) and [GOVERNANCE](https://github.com/organicmaps/organicmaps/blob/master/docs/GOVERNANCE.md) up to date? Aside from the documentation, there hasn't been much said on either of them.

No, they are out of date. I filed a ticket to update them. What information do you expect to see there?

**Reply by TheAdventurer64** [reply]
What I meant by this is "Are they actually used?" I haven't seen milestones used (aside from the beta stuff) and there doesn't seem to be much mention about the board.

**Reply by pastk** [reply]
Linking the ticket
- https://github.com/organicmaps/organicmaps/issues/3348
